pkg/config/spec.go

```diff
@@ -438,34 +438,18 @@ func NewResetSpec(cfg *Config) (*v1.ResetSpec, error) {
 
 	target := ep.State.Disk
 
-	// OEM partition is not a hard requirement
-	if ep.OEM != nil {
-		if ep.OEM.MountPoint == "" {
-			ep.OEM.MountPoint = constants.OEMDir
-		}
-		ep.OEM.Name = constants.OEMPartName
-	} else {
+	// OEM partition is not a hard requirement for reset unless we have the reset oem flag
+	cfg.Logger.Info(litter.Sdump(ep.OEM))
+	if ep.OEM == nil {
 		// We could have oem in lvm which won't appear in ghw list
 		ep.OEM = partitions.GetPartitionViaDM(cfg.Fs, constants.OEMLabel)
 	}
 
-	if ep.OEM == nil {
-		cfg.Logger.Warnf("no OEM partition found")
-	}
-
 	// Persistent partition is not a hard requirement
-	if ep.Persistent != nil {
```

Comment:
removed clutter here and above, not really needed.

Reply:
It's not clear why it's not needed but I trust you :). It seems to be setting the default for some reason.